DMX.Core.Api.Infrastructure.Provision/Brokers/Clouds/CloudBroker.ResourceGroups.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.ResourceManager.Fluent;
+using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+using Microsoft.Azure.Management.ResourceManager.Fluent.Models;
+
+namespace DMX.Core.Api.Infrastructure.Provision.Brokers.Clouds
+{
+    public partial class CloudBroker
+    {
+        public async ValueTask<bool> CheckResourceGroupExistsAsync(string resourceGroupName) =>
+            await this.azure.ResourceGroups.ContainAsync(resourceGroupName);
+
+        public async ValueTask<IResourceGroup> CreateResourceGroupAsync(
+            string resourceGroupName)
+        {
+            return await this.azure.ResourceGroups
+                .Define(resourceGroupName)
+                .WithRegion(Region.USWest)
```

Comment:
Paremeter Aliasing

Reply:
The passed in values match the parameter names in part and remains readable without aliases, so don't think they're necessary here.